catalog-model/src/main/java/org/openecomp/sdc/be/model/operations/impl/GraphLockOperation.java

```diff
@@ -74,14 +67,7 @@ public StorageOperationStatus unlockComponentByName(String name, String componen
 
     @Override
     public StorageOperationStatus lockComponentByName(String name, NodeTypeEnum nodeType) {
-        log.info("lock resource with name {}", name);
-        JanusGraphOperationStatus lockElementStatus = null;
-        try {
-            lockElementStatus = disable ? JanusGraphOperationStatus.OK : janusGraphGenericDao.lockElement(name, nodeType);
-        } catch (Exception e) {
-            lockElementStatus = JanusGraphOperationStatus.ALREADY_LOCKED;
-        }
-        return DaoStatusConverter.convertJanusGraphStatusToStorageStatus(lockElementStatus);
+        return lockComponentDefault(String name, NodeTypeEnum nodeType);
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Syntax error on token &#34;nodeType&#34;, delete this token